app/controllers/provider_interface/provider_interface_controller.rb

```diff
@@ -97,28 +97,15 @@ def redirect_if_setup_required
       if @provider_setup.next_agreement_pending
         redirect_to provider_interface_new_data_sharing_agreement_path
       elsif @provider_setup.next_relationship_pending
-        redirect_to setup_organisation_permissions_path
+        redirect_to provider_interface_organisation_permissions_setup_index_path
       end
     end
 
     def performing_provider_organisation_setup?
-      setup_controllers = [ProviderInterface::ProviderAgreementsController]
-
-      setup_controllers << if FeatureFlag.active?(:accredited_provider_setting_permissions)
-                             ProviderInterface::OrganisationPermissionsSetupController
-                           else
-                             ProviderInterface::ProviderRelationshipPermissionsSetupController
-                           end
-
-      setup_controllers.include?(request.controller_class)
-    end
-
-    def setup_organisation_permissions_path
-      if FeatureFlag.active?(:accredited_provider_setting_permissions)
-        provider_interface_organisation_permissions_setup_index_path
-      else
-        provider_interface_provider_relationship_permissions_organisations_path
-      end
+      [
+        ProviderInterface::ProviderAgreementsController,
+        ProviderInterface::OrganisationPermissionsSetupController,
+      ].include?(request.controller_class)
```

Comment:
Just wanted to double check what this method is doing!

Reply:
This method is checking whether the user is already in the setup flow. If they are not, and setup is required, then we redirect them to the flow. If they are already in the flow, then they are not redirected to the start

Reply:
Cheers, haven't seen the syntax before